config/locales/en.yml

```diff
@@ -21,14 +21,24 @@
 
 en:
   project:
-    title: TaDa
+    title: ToDo
   list:
-    all: All to-do lists
+    all: All ToDo lists
+    new:
+      title: New ToDo List
+      name_field: NAME
+      description_field: DESCRIPTION
+      notice: Create the list and start adding a new ToDo items.
+      create_button: Create New ToDo List
+      submit_button: CREATE LIST
+      cancel_button: CANCEL
```

Comment:
@vlasiak, Uppercase is a part of styles. Let's abstract the styles from the texts, in order developers do not have to change locals when there is a need to change UI:
- change uppercase into capitalized, for example `CREATE LIST` => `Create List`,
- supply proper css styles.